src/test/java/com/hazelcast/aws/impl/DescribeInstancesTest.java

```diff
@@ -88,7 +84,8 @@ public void test_whenAccessKey_And_IamTaskRoleEnvVar_Null_With_DefaultRole_Assig
         DescribeInstances descriptor = spy(new DescribeInstances(awsConfig));
         doReturn(defaultIamRoleName).when(descriptor).retrieveRoleFromURI(uri);
         doReturn(someDummyIamRole).when(descriptor).retrieveRoleFromURI(roleUri);
-        descriptor.checkKeysFromIamRoles(mockedEnv);
+        doReturn(mockedEnv).when(descriptor).getEnvironment();
```

Comment:
`checkKeysFromIamRoles` no longer accepts `Environment`, instead it creates when it's necessary. So `getEnvironment` method is added to be able to support mocking this on-demand initiation.